packages/bpk-tokens/tokens/base.ios.json

```diff
@@ -35,6 +35,48 @@
       "originalValue": "{!WHITE}",
       "name": "backgroundLightColor"
     },
+    {
+      "type": "color",
+      "category": "colors",
+      "value": "#000000ff",
+      "originalValue": "{!BLACK}",
+      "name": "backgroundPrimaryDarkColor"
```

Comment:
I think that for consistency with Apple we should call the `backgroundPrimaryColor` just `backgroundColor`
Ref: https://developer.apple.com/documentation/uikit/uicolor/ui_element_colors

Reply:
Ok cool.
What do you think about `textPrimaryColor`?

Reply:
Also just thinking... What happens in swift if someone uses `Color.backgroundColor`. Would they get Apple's, or ours?

Reply:
Apple's colours are named differently, so no clash would occur!!
https://developer.apple.com/documentation/uikit/uicolor/ui_element_colors?language=objc